Description: Do not run toplevel tests
 The toplevel tests are old namespaced stuff, which we don't really need to
 run and which are breaking in Python 3. This patch makes it so that testr only
 search in the non-namespaced folder.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2015-04-19

--- python-oslo.i18n-1.5.0.orig/.testr.conf
+++ python-oslo.i18n-1.5.0/.testr.conf
@@ -2,6 +2,6 @@
 test_command=OS_STDOUT_CAPTURE=${OS_STDOUT_CAPTURE:-1} \
              OS_STDERR_CAPTURE=${OS_STDERR_CAPTURE:-1} \
              OS_TEST_TIMEOUT=${OS_TEST_TIMEOUT:-60} \
-             ${PYTHON:-python} -m subunit.run discover -t ./ . $LISTOPT $IDOPTION
+             ${PYTHON:-python} -m subunit.run discover -t ./ ./oslo_i18n $LISTOPT $IDOPTION
 test_id_option=--load-list $IDFILE
 test_list_option=--list
\ No newline at end of file
